Packs/PAN-OS/ReleaseNotes/2_5_1.md

```diff
@@ -0,0 +1,6 @@
+#### Integrations
+
+##### Palo Alto Networks PAN-OS
+
+- Added a new command to get the certificate information from Panorama or firewall as seen under Certificate Management:
```

Comment:
Added a new command to get the certificate information from Panorama or a firewall, as seen under Certificate Management: